js/apps/system/_admin/aardvark/APP/react/src/views/views/Components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from "react";
+import AccordionItem from "./AccordionItem";
+import { IAccordionItemViewArgs, IAccordionViewArgs } from "./types";
+import "./styles.css";
+
+const AccordionView = ({
+  accordionConfig = [],
+  allowMultipleOpen
+}: IAccordionViewArgs) => {
+  const [activeIndices, setActiveIndices] = useState<Array<number>>([]);
+
+  useEffect(() => {
+    const defaultActiveTab = accordionConfig.filter(
+      (config) => config.defaultActive
+    );
+    if (defaultActiveTab.length) {
+      setActiveIndices([defaultActiveTab[0].index]);
+    }
+  }, []);
+
+  const handleClick = (index: number) => {
+    const doesIndexExist = !!activeIndices.filter((val) => val === index)
+      .length;
+
+    if (allowMultipleOpen) {
+      if (doesIndexExist) {
+        const updatedIndices = activeIndices.filter((val) => val !== index);
+        setActiveIndices([...updatedIndices]);
+      } else {
+        setActiveIndices([...activeIndices, index]);
+      }
+    } else {
+      if (doesIndexExist) {
+        setActiveIndices([]);
+      } else {
+        setActiveIndices([index]);
+      }
+    }
+  };
+
+  if (!accordionConfig.length || !accordionConfig) {
```

Comment:
The operands around `||` should be reversed.

Reply:
Resolved.